dots.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>Dot-calling returns pixels that are enriched relative to some local background. Such pixels often come in groups ("clusters"). By default <code>dots()</code> picks a single representative for each cluster (i.e. centroid). However, cooltools users can easily turn clustering off for debugging or alternative clustering approaches:</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>